e/envoy/build_info.json

```diff
@@ -0,0 +1,16 @@
+{
+    "package_name" : "envoy",
+    "github_url": "https://github.com/envoyproxy/envoy",
+    "version": "v1.25.3",
+    "package_dir": "e/envoy",
+    "default_branch": "main",
+    "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
+    "build_script": "envoy_1.25.3_ubi8.4.sh",
```

Comment:
Also @sumitd2  Can you make changes in build script which are aligned to currency need. Handling failures for build and test with different exit codes

Reply:
Oh i forgot

Reply:
done